vercel-deployment-task-source/task.json

```diff
@@ -10,8 +10,8 @@
   "author": "Vercel",
   "version": {
     "Major": 1,
-    "Minor": 5,
-    "Patch": 1
+    "Minor": 3,
```

Comment:
Also applies here – should be higher as the current latest

Reply:
Bumped to `1.6.0` in https://github.com/vercel/vercel-azure-devops-extension/pull/33/commits/934a7837e67ab25e7dc680781091b190ed371785

Reply:
BTW I think reverting #27 would've been good enough

Reply:
Thanks for the note 👍 

I couldn't tell at a glance if https://github.com/vercel/vercel-azure-devops-extension/pull/30 built on top of changes in https://github.com/vercel/vercel-azure-devops-extension/pull/27.

My intention here is do the closest thing to a "safe rollback" of recent changes since users are impacted. After which we'll bring back any changes that were unrelated to the issue.